src/transactions/SignatureChecker.cpp

```diff
@@ -18,9 +18,10 @@ using xdr::operator<;
 using xdr::operator==;
 
 SignatureChecker::SignatureChecker(
+    bool alwaysOk,
     Hash const& contentsHash,
     xdr::xvector<DecoratedSignature, 20> const& signatures)
-    : mContentsHash(contentsHash), mSignatures(signatures)
+    : mAlwaysOk{alwaysOk}, mContentsHash{contentsHash}, mSignatures{signatures}
```

Comment:
I don't think adding a bool like this is the right approach: if we make any changes to the way we check signatures, we'll need the protocol version, so I'd rather have the protocol version passed in.